operator/cmd/mesh/operator-init.go

```diff
@@ -51,7 +51,7 @@ func addOperatorInitFlags(cmd *cobra.Command, args *operatorInitArgs) {
 	cmd.PersistentFlags().StringVar(&args.common.operatorNamespace, "operatorNamespace", operatorDefaultNamespace,
 		"The namespace the operator controller is installed into")
 	cmd.PersistentFlags().StringVar(&args.common.istioNamespace, "istioNamespace", istioDefaultNamespace,
-		"The namespace Istio is installed into")
+		"The namespaces the operator controller watches")
```

Comment:
Is there any way we can tell users that they can pass multiple namespaces to watch?

Reply:
Added more comments and an example.